README.md

```diff
@@ -1,3 +1,167 @@
 [![Travis](https://img.shields.io/travis/triplea-game/dice-server-js.svg?style=flat-square)](https://travis-ci.org/triplea-game/dice-server-js)
 # dice-server-js
-The new TripleA dice server powered by Node.js
+The new TripleA dice server re-written in JavaScript.
+
+## Setup
+### Prequisites
+In order to be able to run the server you need to have a couple of things installed on your system:
+ - `node.js` >= 10
+ - `yarn`
+ - `postgres` >= 10
+After those are installed on your system you can follow the steps below to get yourself up and running.
+The steps assume node and yarn are on the `PATH` of your system, if that's not the case, you need to replace `node` and `yarn` with the full qualified path name to the executable.
+
+### One-Time-Setup
+1. Clone this Repository in a directory of your choice.
+2. Open a terminal and navigate to the newly checked out directory.
+3. Run `yarn install`, this will resolve all dependencies for you and install them in `node_modules`
+4. - Generate an RSA key-pair, if you have installed `openssl` and it's on your PATH, you can simply run `yarn generate-keypair`, which will create the files `privkey.pem` and `pubkey.pem` in the current directory.
+   - On productive systems you should move them into a "save directory" and (at least on UNIX-like systems) restrict read access of the private key to the user that's going to run the server and root.
+5. - Create a file `config.json` which contains all information the server needs to start.
+   - Alternatively those configuration options can be passed via CLI directly or as environment variables, check the [nconf docs](https://github.com/indexzero/nconf#example) for more information.
+   - The config files' Layout will be explained below.
+6. Create a Database in postgres to be used by the Server.
+   
+#### config.json
+```json
+{
+  "port": 7654,
+  "database": {
+    "username": "postgres",
+    "password": "",
+    "host": "localhost",
+    "port": 5432,
+    "database": "dicedb"
+  },
+  "email": {
+    "smtp": {
+      "host": "smtp.provider.com",
+      "port": 587,
+      "auth": {
+        "user": "your.email@provider.com",
+        "pass": "super secret password no one will ever guess"
+      }
+    },
+    "display": {
+      "sender": "\"Display Name\" <your.email@provider.com>",
+      "server": {
+        "protocol": "http",
+        "host": "localhost",
+        "port": 7654,
+        "baseurl": ""
+      }
+    }
+  },
+  "keys": {
+    "private": "./privkey.pem",
+    "public": "./pubkey.pem"
+  }
+}
+```
+- `port`: The port node.js will listen on. Required.
+- `database`: Details about the database connection. Required.
+   - `username`: Username to authenticate with the Database. Default: `postgres`.
+   - `password`: Password to authenticate with the Database. Default ` `.
```

Comment:
You have the default DB password listed as a single space here, whereas the above configuration looks like it's an empty string.  You may just want to say "Default: (empty)" or something like that.

Reply:
Yeah, unfortunately I can simply put an empty string into a code block using markdown.
I thought nobody would tell the difference just by looking.

Reply:
I'm going for the latter option